tcms/testruns/static/testruns/js/get.js

```diff
@@ -52,6 +67,17 @@ $(document).ready(() => {
     })
 })
 
+function selectedExecutions() {
+    const allSelected = $('.test-execution-checkbox:checked')
+
+    if (!allSelected.length) {
+        const warningText = $('#test_run_pk').data('trans-no-executions-selected')
+        alert(warningText)
+        return { hasSelected: false, selection: [] }
+    }
+
+    return { hasSelected: true, selection: allSelected }
```

Comment:
we could delete line 76 and make this `return { hasSelected: allSelected.length !== 0, selection: allSelected }`

I don't have any preference

Reply:
I'm not seeing this piece of code so either clarify or mark it as resolved if this is not an issue anymore.